linode/manager
# fix: [M3-7625] – Update VPC docs links with published docs

## Description 📝
Update several placeholder links with the published VPC docs links

## Changes  🔄
- Update links to VPC docs
- Organize `constants.ts` file better

## How to test 🧪
### Verification steps 
Confirm that:
1. The “Docs” link in the entity header on the VPC landing and VPC Detail pages point at  https://www.linode.com/docs/products/networking/vpc/
2. The “Getting Started” link in the entity header on the VPC Create page points at https://www.linode.com/docs/products/networking/vpc/get-started/
3. When trying to assign a linode with multiple configuration profiles, the "Learn More" link points at https://www.linode.com/docs/products/compute/compute-instances/guides/configuration-profiles/
4. There are no broken docs links throughout the VPC feature

## As an Author I have considered 🤔
